File: fixtures/adapter-php/basic/Elsewhere.php
<?php
// UserService is declared in another file. Tier 2 does not resolve across
// files, so this reference records no cross-file edge — and `status` reports
// the tier so nobody mistakes that silence for "nothing uses UserService".

namespace App;

use App\Service\UserService;

class Elsewhere
{
    public function run(): string
    {
        $service = new UserService();
        return $service->handle();
    }
}

File: fixtures/adapter-swift/basic/Elsewhere.swift
// UserService is declared in another file. Swift has no import statement for a
// file in the same target — the name is simply in scope — so there is nothing
// here for a resolver to follow even in principle, and Tier 2 records no
// cross-file edge. `status` reports the tier so that silence is not mistaken
// for "nothing uses UserService".

struct Elsewhere {
    func run() -> String {
        let service = UserService()
        return service.handle()
    }
}
